gui/src/components/Spinner.svelte

```diff
@@ -1,4 +1,8 @@
-<div />
+<script lang="ts">
+  export let inline: boolean = false;
```

Comment:
nit: you can use inference on the type here

```suggestion
  export let inline = false;
```